test_vm/Vagrantfile

```diff
@@ -5,21 +5,33 @@ require_relative 'vagrant_config_helper'
 
 VAGRANTFILE_API_VERSION = "2"
 
+file_to_disk = "./disk1.vdi"
```

Comment:
Is this path relative to dbfit project folder or to the VirtualBox/vagrant disks?